slashing/program/tests/duplicate_block_proof.rs

```diff
@@ -179,13 +189,14 @@ pub(crate) fn new_rand_shreds<R: Rng>(
 #[tokio::test]
 async fn valid_proof_data() {
     let mut context = program_test().start_with_context().await;
+    setup_clock(&mut context).await;
 
     let authority = Keypair::new();
     let account = Keypair::new();
 
     let mut rng = rand::thread_rng();
     let leader = Arc::new(Keypair::new());
-    let (slot, parent_slot, reference_tick, version) = (53084024, 53084023, 0, 0);
+    let (slot, parent_slot, reference_tick, version) = (SLOT, 53084023, 0, 0);
```

Comment:
nit: same here `SLOT - 1` is a bit easier to read for the parent slot for all the tests in this file

Reply:
reorganized into test constants here 880d9c0e784174a65d3fc9a8530905455db0de22